src/components/atoms/video/Video.tsx

```diff
@@ -1,18 +1,22 @@
-import React from "react";
-import { CardMedia } from "@mui/material";
+import React from 'react'
+import { CardMedia } from '@mui/material'
 
 type VideoProps = {
-  url: string;
-};
+  url: string
+}
 
 // WIP
 export function Video({ url }: VideoProps) {
   return (
-    <CardMedia component={"iframe"} src={addQueryParamsToVideoUrl(url)} sx={{ aspectRatio: "16/9" }} />
-  );
+    <CardMedia
+      component={'iframe'}
+      src={addQueryParamsToVideoUrl(url)}
+      sx={{ aspectRatio: '16/9' }}
+    />
```

Comment:
What's the number of props for a new line here? Sometimes it's unnecessary, but sometimes it's way better for legibility. I don't have a thought, I just don't want to think about it for only a couple of props.

Reply:
I'm not seeing options for this. Maybe there's a better solution than prettier that gives us more control